### ISSUE TITLE
Stop working since Aug 29 - 2023

### ISSUE THREAD

**Issue by kp2067** [CRutkowski/Kijiji-Scraper#38]
Kijiji just updated something on their website and the script is now can not find new ads with any URL.  Please help

**Comment by YannGagnon** [comment]
I can corroborate, this just stopped working for me also.

**Comment by YannGagnon** [comment]
> Kijiji just updated something on their website and the script is now can not find new ads with any URL. Please help

Kijiji changed all the div tags. Many are alpha-numeric strings instead of being human-readable.

**Comment by YannGagnon** [comment]
We can talk about it, let me know how to reach you.  It's even worst than I thought though, the sponsored and top ads are now labelled dynamically with javascript, so you can't differentiate them with beautifulsoup anymore.

**Comment by closewait** [comment]
I did some quick and dirty changes with minimal testing in this fork: https://github.com/closewait/Kijiji-Scraper.
I'll do a pull request after more testing.

**Comment by Ankit-Mehra** [comment]
yes it stopped working sometime back in august. Did some changes and its working since last one and half month. Will make a pull request soon

**Comment by rezafilsoof1** [comment]
@Ankit-Mehra - any updates on the PR? Can I help in any way?
